JesusOmarR/react-apprenticeship-capstone1
# Hooks challenge

## Description
```
Added the VideoDetails page to display information of the selected video and 
a list of related videos, this was made for achieve the third react mini challenge






## How should this be tested?
Run All the tests that were made and verify if the components of the Video Details are working correctly

## What is the deliverable?
```
React Third mini challenge
```
## Screenshots and links
```
<img width="1318" alt="image" src="https://user-images.githubusercontent.com/97203630/149475098-4b8eb2e8-99c7-4369-b12e-2a170ba1aa64.png">

![image](https://user-images.githubusercontent.com/97203630/149846449-f3fe3cbe-7ff6-41ac-8927-b1d313d2ed8b.png)


```

## Checklist
- [ x ] I added the necessary documentation, if appropriate.
- [ x ] I have commented my code, particularly in hard-to-understand areas
